lldb/source/Plugins/SymbolFile/NativePDB/SymbolFileNativePDB.cpp

```diff
@@ -365,18 +365,20 @@ void SymbolFileNativePDB::InitializeObject() {
 }
 
 uint32_t SymbolFileNativePDB::CalculateNumCompileUnits() {
+  if (m_cu_count)
+    return *m_cu_count;
```

Comment:
`CalculateNumCompileUnits` is wrapped by `GetNumCompileUnits`, which makes sure the computation happens only once. I think you can delete this variable and just call `GetNumCompileUnits` everywhere.